subprojects/enterprise/src/main/java/org/gradle/internal/tasks/TaskProperties.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.internal.tasks;
+
+import java.util.Set;
+
+public interface TaskProperties {
+
+    Set<InputFileProperty> getInputFiles();
```

Comment:
We typically name these things like `getFileProperties()` in other places because a) you are not returning the files themselves, and b) the returned file system objects can be files, directories, collections of either etc.

Reply:
Done in b302852b1099f60dfa2a91f531bc7ece728bd797.